### ISSUE TITLE
manage user roles

### ISSUE THREAD

**Issue by isabel-amon** [hpi-swt2/vm-portal#110]
**Beschreibung**
As an admin I want to be able to manage the rights of the users of our tool, so that everyone can use the functionalities  he/she needs. On a all-users page I want to see a grouped toggle button (see screenshot below) with the options of user, employee and admin. The current role of the user should be pressed down so I can see it clearly. 
When I click on a role the user not currently has, I want his or her role changed accordingly.

<img width="217" alt="screen shot 2018-12-19 at 2 27 16 pm" src="https://user-images.githubusercontent.com/5675363/50222993-3e810300-039a-11e9-97aa-095236d2166a.png">

I only want to be able to change my own role, if there is still one other admin left after I do so, in order to avoid there being no admin left. In this case the other options next to my name in my grouped button should be greyed out and not clickable.

**Akzeptanzkriterien**
- [x] A grouped button exists on the all-users-page, displaying the users current role
- [x] This button is only visible for admins
- [x] Pressing another role changes the role of the according user
- [x] After the change happened the user gets access to the tool according to his new role
- [x] An admin can only change his/her own role if ensured that there is at least one other admin left
- [x] If no other admin would be left, the last admins other role buttons are not clickable

**Priorität**

**Aufwandsschätzung**

**Comment by skydivin4ng3l** [comment]
depends on #77

**Comment by lassesteffen** [comment]
- [ ] add a button group to the users page in a role column
- [ ] show selected role
- [ ] make this only visible for admins
- [ ] make the role changeable by clicking on the button
- [ ] grey out button options for the admin if there is only one admin
- [ ] make the buttons unclickable in the former case

**Comment by lassesteffen** [comment]
> Before an employee is degraded to user, there is a check making sure no vm is left without any responsible people

@isabel-amon is this already implemented? I can not see it being implemented and this is the only thing keeping from closing this issue. If it possible to move this into a seperate issue, so we can deploy this feature?

**Comment by isabel-amon** [comment]
sorry for the late answer, since this issue is closed now is everything clear? this was not implemented and should be implemented by you, I can move it into a separate issue if you still like.

**Comment by lassesteffen** [comment]
It was not done, wasnt careful with github references. You can see in my task list, that the last two options are not taken care of right now. I just thought we might be able to move them in a new issue, so we can actually close this one.
